### PR TITLE
Interface update

### PR DESCRIPTION
**Pull request by joshurtree**
Changes player tools into seperate pages for each tool making them directly accessable from the top menu. Also added a menu for updating profile to top menu.

### CODE REVIEW

File: src/pages/playertools.tsx [hunk 1]
```diff
@@ -33,8 +83,9 @@ const PlayerToolsPage = () => {
 	const [eventData, setEventData] = useStateWithStorage('tools/eventData', undefined);
 	const [activeCrew, setActiveCrew] = useStateWithStorage('tools/activeCrew', undefined);
 
+	const urlParams = new URLSearchParams(window.location.search);
```
Comment (AlexCPU) [inline src/pages/playertools.tsx:86]: [website.e0ab816a-7a0d-494f-9ef4-2605d4872e96.log](https://github.com/stt-datacore/website/files/7143560/website.e0ab816a-7a0d-494f-9ef4-2605d4872e96.log)
This is causing a build failure here.

**Comment by AlexCPU** [comment]
This is still failing build on both Cloudflare Pages (preview builds) and Github Actions (production build).

1. Have you managed to get this to build locally?
2. Can you see the build log for the Github actions build?  (I know you won't be able to get to the cloudflare pages build log)

File: src/pages/playertools.tsx [hunk 2]
```diff
@@ -36,6 +86,13 @@ const PlayerToolsPage = () => {
 	const [dataSource, setDataSource] = React.useState(undefined);
 	const [showForm, setShowForm] = React.useState(false);
 
+	React.useEffect(() => {
+		const urlParams = new URLSearchParams(window.location.search);
+		setShowForm(urlParams.has('update'));
+		if (urlParams.has('clear'))
+			clearPlayerData();
+	}, [window.location.search]);
```
Comment (AlexCPU) [inline src/pages/playertools.tsx:94]: error "window" is not available during server side rendering.

**Comment by ussjohnjay** [comment]
Quick question. Can you re-add `allCrew` as a parameter to the Event Planner instance in `playertools.tsx`? Some of the event planner functions won't work without `allCrew` being properly passed down from the main player tools page (and then you wouldn't need to test for `allCrew' in `eventplanner.tsx`). Thanks!

**Comment by joshurtree** [comment]
Added. Thanks for pointing that out.

**Review by AlexCPU — APPROVED** [review]
LGTM

**Pull request merged**